x/campaign/keeper/campaign_mainnet_test.go

```diff
@@ -0,0 +1,67 @@
+package keeper_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	testkeeper "github.com/tendermint/spn/testutil/keeper"
+	"github.com/tendermint/spn/testutil/sample"
+)
+
+func TestIsCampaignMainnetLaunchTriggered(t *testing.T) {
```

Comment:
I believe we are missing a test case where `MainnetInitialized == false` where the method must return `false`